modules/javafx.graphics/src/main/java/javafx/css/Match.java

```diff
@@ -41,15 +40,15 @@
 public final class Match implements Comparable<Match> {
 
     final Selector selector;
-    final PseudoClassState pseudoClasses;
+    final Set<PseudoClass> pseudoClasses;
```

Comment:
Should this be an `ObservableSet`? Changing the type to the `Set` superclass will mean that applications would need to do an `instanceof` check to know whether it was observable or not?

Reply:
Btw, I meant to add this comment on the public getter method.